source/Engine.cpp

```diff
@@ -1008,12 +1008,15 @@ void Engine::Draw() const
 		Point pos = it.position * zoom;
 		double radius = it.radius * zoom;
 		if(it.outer > 0.)
-			RingShader::Draw(pos, radius + 3., 1.5f, it.outer, color[it.type], 0.f, it.angle);
+			RingShader::Draw(pos, radius + 3., 1.5f, it.outer,
+				Color::Multiply(it.alpha, color[it.type]), 0.f, it.angle);
 		double dashes = (it.type >= 3) ? 0. : 20. * min(1., zoom);
```

Comment:
Oh no, this is out of date.
```suggestion
		double dashes = (it.type >= 5) ? 0. : 20. * min(1., zoom);
```